src/service/quizService.ts

```diff
@@ -2,7 +2,8 @@ import { QuizPayload, QuizResponse } from "@models/quiz";
 import axios from "axios";
 import axiosRetry from "axios-retry";
 
-const API_URL = "https://opentdb.com/api.php";
+// TODO: 네트워크 에러 발생 유도
+const API_URL = "https://opentdb.com/api.php" + "error";
```

Comment:
```suggestion
const API_URL = "https://opentdb.com/api.php";
```

❗️ 테스트 발생 유도를 위한 작업입니다.